recipe/meta.yaml

```diff
@@ -40,6 +40,8 @@ requirements:
     - r-rcpp >=0.12.12
     - r-systemfonts >=0.1.1
     - cairo
+    # cairo pulls in "libglib" but only "glib" carries GLib's pkg-config files needed here
+    - glib
```

Comment:
Should inspect conda-build's log to see if this introduces overlinking. If yes, we should add a `ignore_run_exports_from` for `glib`.